x-pack/plugin/relevance-search/src/main/java/org/elasticsearch/xpack/relevancesearch/query/CurationQueryRewriter.java

```diff
@@ -27,8 +27,7 @@ class CurationQueryRewriter extends AbstractQueryRewriter<CurationsGroup> {
 
     @Override
     protected String getSettingsId(SearchEngine searchEngine) {
-        // TODO: add curation to engine.
-        return null;
+        return searchEngine.getCurations();
```

Comment:
Since we renamed the parameters for example relevanceSettingsId -> relevanceSettings should this be refactored to getSettings()?